compaction/compaction_manager.cc

```diff
@@ -12,8 +12,10 @@
 #include "sstables/sstables.hh"
 #include "sstables/sstables_manager.hh"
 #include "replica/database.hh"
+#include <optional>
 #include <seastar/core/metrics.hh>
 #include <seastar/core/coroutine.hh>
+#include <seastar/core/future.hh>
```

Comment:
nit: I doubt this and the <optional> include was really needed here, they are probably included indirectly already. Only add new includes if the file doesn't compile.